server/src/helpers/following.js

```diff
@@ -79,14 +79,14 @@ class FollowingHelper {
       if (!user) {
         return 'notAUser';
       }
-      const checkFollowing = await user.countFollower({ followerId });
+      const checkFollowing = await user.hasFollower(followerId);
       const deleteFollowing = await user.removeFollower(followerId);
-      if (checkFollowing === 1 && deleteFollowing) {
```

Comment:
Also, from my understanding, this looks like a reactor, if yes, please specify it on the PR comment section so everyone will be aware of the changes made that are not directly related to the notification task. I suggest you put it under the **Background information section**